test/CoveragePool.test.js

```diff
@@ -263,10 +268,37 @@ describe("CoveragePool", () => {
       })
     })
 
-    context("when caller is the governance", () => {
-      let tx
+    context("when cancelling risk manager approval process", () => {
+      beforeEach(async () => {
+        await coveragePool
+          .connect(governance)
+          .approveFirstRiskManager(anotherRiskManager.address)
+        await coveragePool
+          .connect(governance)
+          .beginRiskManagerApproval(riskManager.address)
+        tx = await coveragePool
+          .connect(governance)
+          .unapproveRiskManager(riskManager.address)
+      })
 
+      it("should remove timestamp of risk manager", async () => {
```

Comment:
... of risk manager approval
?

Reply:
Done in https://github.com/keep-network/coverage-pools/pull/176/commits/3aa4ecce94e64166f1f4fece8d634156dab0ef1c.